doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -953,7 +954,29 @@ The user that Ray will authenticate with when launching new nodes.
 
     .. tab-item:: Azure
 
-        Not available.
+        A string specifying a comma-separated list of availability zone(s) that nodes may be launched in.
+        This can be specified at the provider level to set defaults for all node types, or at the node level
+        to override the provider setting for specific node types.
+
+        For Azure, availability zone availability depends on each specific VM size / location combination.
+        Node-level configuration in ``available_node_types.<node_type_name>.node_config.azure_arm_parameters.availability_zone``
+        takes precedence over provider-level configuration.
+
+        * **Required:** No
+        * **Importance:** Low
+        * **Type:** String
+        * **Default:** "auto" (let Azure automatically pick zones)
+        * **Example values:**
+          
+          * ``"1,2,3"`` - Use zones 1, 2, and 3
+          * ``"1"`` - Use only zone 1
+          * ``"none"`` - Explicitly disable zones
+          * ``"auto"`` or omit - Let Azure automatically pick zones
```

Comment:
For provider level, omit means auto-pick. What about node level, what does it mean if I omit it: auto-pick or inherit from node level?

Reply:
Node should have higher order precedence. So if there is no provider-level configuration, we set every node to auto, unless a node specifies its own AZ configuration.

I assume the above is the way this is implemented @marosset?

Reply:
yes, node level has higher precedence over the provider level setting.
All of the same values (one or more zones, none, and auto) can be set at either the node or the provider level.
if none are set, you'll get the auto behavior for all nodes